common/oe_host_string.h

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+/*
+ * There are separate OE functions for strerror and strerror_s.
+ * Therefore we suppress the warning about not using more secure CRT functions.
+ */
```

Comment:
nit: Move this down to where oe_strerror is

Good change though, I didn't realize it was only one funciton causing the warning.